### PR TITLE
Antispam

### PR DESCRIPTION
**Pull request by ShiroUsagi-san**
Adding a basic antispam system through an hidden field

### CODE REVIEW

**Comment by Lymkwi** [comment]
*surprised pikachu face*

we'll fix the history later dw

File: insalan/user/tests.py [hunk 1]
```diff
@@ -175,6 +174,39 @@ def send_valid_data(data, check_fields=[]):
...
+
+            send_bot_data(
+                {
+                    "username": "newplayer",
+                    "password": "1111qwer!",
+                    "password_validation": "1111qwer!",
+                    "email": "email@example.com",
+                    "name": "je suis un bot"
+                },
+                [
```
Comment (Lugrim) [inline insalan/user/tests.py:199]: We don't wanna check fields, you can remove that part (and the argument in `send_bot_data`)

Comment (ShiroUsagi-san) [reply insalan/user/tests.py]: solved by [#2f4b2](https://github.com/InsaLan/backend-insalan.fr/pull/75/commits/2f4b2cfb49d91a7d7b973e3c55f704a5774d0a00)

**Review by Lugrim — APPROVED** [review]
Looks good to me, it'll have to be used in the frontend once this one is merged

**Pull request merged**